packages/stk/cmake/Dependencies.cmake

```diff
@@ -11,6 +11,7 @@ SET(SUBPACKAGES_DIRS_CLASSIFICATIONS_OPTREQS
   IO              stk_io               PT  OPTIONAL
   Search          stk_search           PT  OPTIONAL
   Transfer        stk_transfer         PT  OPTIONAL
+  Middle_mesh     stk_middle_mesh      PT  OPTIONAL
```

Comment:
This needs to be declared 'EX' instead of 'PT'.  You can't have a non-`EX` package having a required dependency on an `EX` TPL.